### DISCUSSION TITLE
How can i use this fine tuned model with Whisper CLI? CrisperWhisper

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [openai/whisper#2321] [General]
https://huggingface.co/nyrahealth/CrisperWhisper

I want to use existing whisper CLI but this model is that possible?

e.g. modifying this command

`            string srCommand = @$"cmd /c {gpuIdCommand}whisper ""{srExtractMp3Name}"" --model large-v1 --language en --initial_prompt ""Welcome to our Youtube channel."" --best_of 10 --beam_size 10 --patience 10 --output_dir  ""{srFileDirectory}""";`

**Comment by FurkanGozukara** [comment]
any help appreciated ty so much

@jongwook , @ryanheise , @petterreinholdtsen , @HennerM , @VulumeCode , @guillaumekln , @vickianand , @fcakyon , @jumon , @tomstuart , @wangchou , @cool-RR

**Comment by ryanheise** [comment]
Did you finetune it with huggingface tooling? If so, you likely have a huggingface model and will need to convert it back to a Whisper model. See #830 .

Once you have a native model, you can pass it on the command line with `--model model_path` where `model_path` must be different from one of the original Whisper model names.
